lib/client-assets.php

```diff
@@ -495,25 +497,21 @@ function gutenberg_register_vendor_scripts() {
 		'https://unpkg.com/moment@2.22.1/' . $moment_script,
 		array()
 	);
-	$tinymce_version = '4.7.2';
-	gutenberg_register_vendor_script(
-		'tinymce-latest',
-		'https://unpkg.com/tinymce@' . $tinymce_version . '/tinymce' . $suffix . '.js'
-	);
+	$tinymce_version = '4.7.11';
```

Comment:
Ugh too bad we can't base that on https://github.com/WordPress/wordpress-develop/blob/186bb7c8a908de977934d77fc5633b493ba2223f/src/wp-includes/version.php#L21